### PR TITLE
Fix pack config retrieval

### PR DESCRIPTION
**Pull request by Kami**
Currently this runner manually retrieves pack config inside the runner code.

This is likely very old and legacy approach since for a long time now, runners received config values as a runner constructor argument.

This pull request updates the runner to follow this new config retrieval approach which means the runner code itself now also doesn't depend on the database connection anymore (that was the case with the previous version of the code).

In short - it makes the runner to comply with the current "standard" runner interface.

### CODE REVIEW

**Comment by Kami** [comment]
As a "bonus", I also set up codecov for this repo - https://codecov.io/gh/StackStorm/stackstorm-runner-expect/tree/fix_config_retrieval/expect_runner.
